### ISSUE TITLE
About Melodic on Ubuntu 18.04

### ISSUE THREAD

**Issue by pengzhi1998** [Mayankm96/airsim_ros_client#9]
Hi @Mayankm96 , thank you for your great work!
However, is it possible for this to apply to Ubuntu 18.04 and ROS Melodic?

**Comment by Mayankm96** [comment]
Yes, I think the code should work in Ubuntu 18.04 (i.e. as long as AirSim builds). 

However, I have not updated this repository to work with the new AirSim APIs. I would recommend to check the ROS package that can be found [here](https://github.com/microsoft/AirSim/tree/master/ros/src/airsim_ros_pkgs).

**Comment by pengzhi1998** [comment]
Thank you for your reply! I'll take a look.
